### ISSUE TITLE
Hallux sesamoid injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11185]
**Preferred term label:**
Hallux sesamoid injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
Foot x-ray showing fracture, subluxation, or dislocation of the sesamoid of the hallux
(https://uts.nlm.nih.gov/uts/umls/concept/C2118109 has the fracture part) 
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sesamoid injuries of the hallux

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This is a bundled term and probably closer to a disease entity (etiology, i.e., injury). I realize we may not want to have subatomic resolution with the sesamoid bone, but perhaps it would be better to have "Sesamid fracture" as the term?

**Comment by pnrobinson** [comment]
@MickeySegal 
returning to this after other similar terms, I agree it makes sense as is.

Adding new term: 
- ID: HP:6001403
- label: Hallux sesamoid injury
- parents: [Abnormal hallux morphology (HP:0001844)](https://hpo.jax.org/app/browse/term/HP:0001844)
- pmid: PMID:19232995
- def: Damage affecting one or both of the two small, pea-shaped bones (sesamoids) embedded in the tendons beneath the first metatarsophalangeal joint of the big toe. Foot x-ray shows fracture, subluxation, or dislocation of the sesamoid of the hallux.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11185
- synonyms:
